src/hotspot/share/code/nmethod.hpp

```diff
@@ -621,7 +621,6 @@ class nmethod : public CompiledMethod {
 #if defined(SUPPORT_DATA_STRUCTS)
   // print output in opt build for disassembler library
   void print_relocations()                        PRODUCT_RETURN;
-  void print_pcs() { print_pcs_on(tty); }
```

Comment:
This is a very common pattern, so I was wondering why you got rid of it?

Reply:
There is simply no use of `print_pcs` any more, I changed everything to `print_pcs_on`. Is that ok?

Reply:
It is kind of undoing a well established pattern/convention though. We could replace all `print_foo()` with `print_foo_on(tty)` and so render `print_foo()` unnecessary - but that defeats the whole point of the pattern.

Can't comment on the virtual/non-virtual aspect: not sure what the right/best arrangement is for that.

Anyway, not requesting any changes, just curious about the motivation.